scripts/pre-install

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env bash
 
 SCRIPT_PATH="$(dirname "$0")/$(dirname "$(readlink "$0")")"
-MARATHON_VERSION="1.7.49-f24b03af3"
+MARATHON_VERSION="1.8.143-7c07b88f9"
```

Comment:
I see this log when running `npm install`

```
# Installing raml files 
cp: /var/folders/68/gwtbd7vn5jz7clcv3xjv85p00000gn/T/tmp.aSg8Oeml/marathon-docs-1.8.143-7c07b88f9/docs/rest-api/public/api/v2/schema: No such file or directory
! Updated RAML in /Users/brandonc/m/fe/dcos-ui/src/resources/raml/marathon/v2 
```

I would have also assumed `src/resources/raml/marathon/v2` would have been updated after running it.

Reply:
> I would have also assumed src/resources/raml/marathon/v2 would have been updated after running it.

only if they actually have changed anything

Reply:
I at least see `.marathon-version` having desired `1.8.143-7c07b88f9` :)